### ISSUE TITLE
Ctrl+F to Search doesn't work?

### ISSUE THREAD

**Issue by amiregelz** [glideapps/glide-data-grid#398]
Command+F works (in Mac), but Ctrl+F doesn't seem to work (in Windows).

**Comment by jassmith** [comment]
Works on my computer and on `grid.glideapps.com` whenever the grid has focus. Can you confirm this is not working for you? What web browser are you using? Have you ensured you don't have any other global programs binding that key? Does the browser search box come up instead even if the grid has focus?

**Comment by amiregelz** [comment]
@jassmith Made sure the grid is in focus, yet still not working. 
When the grid has focus, the search box never comes up and nothing happens (the browser search box doesn't show as well).
When the grid doesn't have focus, the Chrome search box comes up.
I am using Google Chrome on Windows 10.

**Comment by jassmith** [comment]
Same for you on grid.glideapps.com?

**Comment by amiregelz** [comment]
@jassmith Yes: https://www.loom.com/share/6a542c3f77bd4697a7ccd42616de3fe1

**Comment by pzcfg** [comment]
If you run and click on this page, then ctrl-F, do you see a "FIND" alert, or the browser search? https://jsfiddle.net/q8tv74wb/show/

I tested a bit on Mac and Windows, and found one issue: if caps lock is on, then this doesn't trigger. It seems this can either be fixed by adding `(event.key === "f" || event.key === "F")` or by using `event.code === "KeyF"` instead.

**Comment by jassmith** [comment]
will fix the event.code thing!
